.editorconfig

```diff
@@ -1,28 +1,12 @@
+# http://editorconfig.org
 root = true
 
 [*]
-indent_style = tab
-indent_size = 4
 charset = utf-8
-trim_trailing_whitespace = true
-insert_final_newline = true
 end_of_line = lf
-
-[*.yml]
-indent_style = space
-indent_size = 2
-
-[*.py]
-indent_style = space
-
-[*.md]
-indent_style = space
-indent_size = 2
-trim_trailing_whitespace = false
-
-[*.sql]
 indent_style = space
-indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
 
-[Dockerfile]
-indent_style = space
+[*.{dm,json}]
```

Comment:
why get rid of all this? it makes somewhat sense to standardize it across the codebase, no?